java/spi.java.hints/src/org/netbeans/modules/java/hints/spiimpl/Utilities.java

```diff
@@ -1352,6 +1354,33 @@ protected JCModifiers modifiersOpt(JCModifiers partial) {
             return super.modifiersOpt(partial);
         }
 
+
+        public JCVariableDecl formalParameter(boolean lambdaParam, boolean recordComponents) {
+            if (token.kind == TokenKind.IDENTIFIER) {
+                if (token.name().startsWith(dollar)) {
+                    com.sun.tools.javac.util.Name name = token.name();
+
+                    Token peeked = S.token(1);
+
+                    if (peeked.kind == TokenKind.COMMA || peeked.kind == TokenKind.RPAREN) {
+                        nextToken();
+                        return JackpotTrees.createVariableWildcard(ctx, name);
+                    }
+                }
+            }
+                        JCTree.JCVariableDecl result = null;
```

Comment:
Nit: note the indent is wrong on this line.
```suggestion
            JCTree.JCVariableDecl result = null;
```